0002-crypto-conditions/0002-crypto-conditions.md

```diff
@@ -110,387 +110,380 @@ informative:
 
 --- note_Feedback
 
-This specification is a part of the [Interledger Protocol](https://interledger.org/) work. Feedback related to this specification should be sent to <public-interledger@w3.org>.
+This specification is a part of the [Interledger Protocol](https://interledger.org/) work. Feedback related to this specification should be sent to <ledger@ietf.org>.
 
 --- abstract
 
-Crypto-conditions provide a mechanism to describe a signed message such that multiple actors in a distributed system can all verify the same signed message and agree on whether it matches the description. This provides a useful primitive for event-based systems that are distributed on the Internet since we can describe events in a standard deterministic manner (represented by signed messages) and therefore define generic authenticated event handlers.
+Crypto-conditions defines a set of encoding formats and data structures for conditions and fulfillments.  A condition uniquely identifies a boolean circuit constructed from one or more logic gates, evaluated by either validating a cryptographic signature or verifying the preimage of a hash digest. A fulfillment is a data structure encoding one or more cryptographic signatures and hash digest preimages that can be used to evaluate the result of the circuit.
```

Comment:
Yeah, the abstract is a bit self-referential:

> Crypto-**conditions** defines a set of encoding formats and data structures for **conditions** and fulfillments.

Maybe we can consolidate these sections, they seem fairly redundant. Maybe move the first paragraph of the intro to the abstract and make the intro more of a section about why we are doing this, perhaps referencing Chris Allen's Smart Signatures etc.